.github/config/ubuntu-22.04/conan/profiles/default

```diff
@@ -1,11 +1,11 @@
 [settings]
+os=Linux
 arch=x86_64
-build_type=Release
+build_type=RelWithDebInfo
```

Comment:
What is the reason for changing this?

Reply:
Can you explain [this](https://github.com/opendocument-app/OpenDocument.droid/blob/main/app/build.gradle#L30C1-L32C61)?
```
"-s", "build_type=Release",
"-s", "&:build_type=RelWithDebInfo",
"-s", "odrcore/*:build_type=RelWithDebInfo",
```

I assume this means a request to build everything as Release, but only odr.core as RelWithDebInfo ? I assume that if we build it like that, there will be less debug info in dependencies. Most of pdf crashes happens not in odr.core, but somewhere deep in libpng, called by either fontforge or poppler, because pdf2htmlEX supplied the wrong argument somewhere. I assume Release instead of RelWithDebInfo for deep dependencies would give less info during crashes.

Reply:
Yes my intention here was to only have debug symbols for `odr.core` and rely on other packages to be stable. That paradigm might shift with `pdf2htmlEX`.

I think I was also facing some multi config issues with Android.

Ultimately this should not necessarily drive our decision here. I think `Release` and `RelWithDebInfo` is fine while `Debug` might be a bit too big.

Reply:
`Debug` is completely unoptimized, let's leave `RelWithDebInfo`. We could also test if `RelWithDebInfo` actually produces binaries that are slower or larger than `Release`